src/app/_mermaid_common/libs/controllers/more-info-modal.ctrl.js

```diff
@@ -0,0 +1,21 @@
+angular.module('mermaid.libs').controller('MoreInfoModalCtrl', [
```

Comment:
Can we name this and the html something specific to data sharing -- or abstract this to handle future "more info" modals elsewhere on the site? We may very well need to show "more info" somewhere else....